core-api/src/main/java/com/optimizely/ab/internal/ConditionUtils.java

```diff
@@ -1,6 +1,6 @@
 /**
  *
- *    Copyright 2018-2019,2021, Optimizely and contributors
+ *    Copyright 2018-2019, 2021, Optimizely and contributors
```

Comment:
Please don't make whitespace changes in areas such as this. Copyright can be standardized later. Thanks!

Reply:
Thanks! I'm more interested in process than this particular change.